go.mod

```diff
@@ -32,6 +32,7 @@ require (
 	github.com/k8snetworkplumbingwg/network-attachment-definition-client v1.3.0
 	github.com/k8snetworkplumbingwg/sriov-cni v2.1.0+incompatible
 	github.com/kevinburke/ssh_config v1.2.0
+	github.com/kr/pretty v0.3.1
```

Comment:
there are plenty of already imported packages can do it, no need to add a new dependency just for this, for example `k8s.io/apimachinery/pkg/util/dump`

Reply:
OK, will update this. Thank you !

Reply:
@tnqn after some tests, it seems we need to bump the apimachinery packge to at least v0.28+ to have the `dump` package, and it would require some api changes in antrea code. I would prefer to use the origin solution.  or the pacakge apimachinery used itself:  `"github.com/davecgh/go-spew/spew`